src/styles/blocks/catalog.scss

```
@@ -0,0 +1,15 @@
+.catalog {
+  padding: 50px 0;

```

```suggestion
  padding: 50px 0 24px;
```
там немає знизу такого великого відступу